lib.php

```diff
@@ -618,18 +618,17 @@ function tincanlaunch_settings($tincanactivityid){
     if(!use_global_lrs_settings($tincanactivityid)){
         $activitysettings = $DB->get_record('tincanlaunch_lrs', array('tincanlaunchid'=>$tincanactivityid), $fields='*', $strictness=IGNORE_MISSING);
         $expresult['tincanlaunchlrsendpoint'] = $activitysettings->lrsendpoint;
-        $expresult['tincanlaunchlrauthentication'] = $activitysettings->lrsauthentication;
+        $expresult['tincanlaunchlrsauthentication'] = $activitysettings->lrsauthentication;
         $expresult['tincanlaunchlrslogin'] = $activitysettings->lrslogin;
         $expresult['tincanlaunchlrspass'] = $activitysettings->lrspass;
         $expresult['tincanlaunchlrsduration'] = $activitysettings->lrsduration;
-        //TODO: remove version when it has been set as global
-        $expresult['tincanlaunchlrsversion'] = '1.0.0';
     }else{//use global lrs settings
         $result = $DB->get_records('config_plugins', array('plugin' =>'tincanlaunch'));
         foreach($result as $value){
             $expresult[$value->name] = $value->value;
         }
     }
+    $expresult['tincanlaunchlrsversion'] = '1.0.0';
```

Comment:
Yes, this is the only place where the version can be set and is no longer a configurable setting within the UI.